modules/oadp-1-4-2-release-notes.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-1-4-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="oadp-1-4-2-release-notes_{context}"]
+= OADP 1.4.2 release notes
+
+The {oadp-first} 1.4.2 release notes lists new features, resolved issues and bugs, and known issues.
+
+[id="new-features-1-4-2_{context}"]
+== New features
+
+.Backing up different volumes in the same namespace by using the VolumePolicy feature is now possible
+
+With this release, Velero now provides resource policies to back up different volumes in the same namespace by using the `VolumePolicy` feature. The supported `VolumePolicy` feature to back up different volumes includes `skip`, `snapshot`, and `fs-backup` actions.
+link:https://issues.redhat.com/browse/OADP-1071[OADP-1071]
+
+.File system backup and data mover can now use short-term credentials
+
+File system backup and data mover can now use short-term credentials such as {aws-short} {sts-first} and {gcp-short} WIF. With this support, backup successfully completes without any `PartiallyFailed` status.
```

Comment:
```suggestion
File system backup and data mover can now use short-term credentials such as {aws-short} {sts-first} and {gcp-short} WIF. With this support, backup is successfully completed without any `PartiallyFailed` status.
```
In this case, passive voice is acceptable because the system is performing the act. That is, the system is completing the backup. For examples of where the passive voice is acceptable, see the ISG: https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#voice